client/app/components/ParameterValueInput.jsx

```diff
@@ -128,31 +130,41 @@ export class ParameterValueInput extends React.Component {
   }
 
   renderEnumInput() {
-    const { value, onSelect, enumOptions } = this.props;
+    const { value, onSelect, enumOptions, allowMultipleValues } = this.props;
     const enumOptionsArray = enumOptions.split('\n').filter(v => v !== '');
     return (
       <Select
         className={this.props.className}
+        mode={allowMultipleValues ? 'multiple' : 'default'}
```

Comment:
Why is the boolean2string conversion needed here? Why not simply pass `allowMultipleValues`?

Reply:
My bad, I didn't notice it was an Antd component restriction.